packages/design/scripts/removeFoundationTempFiles.sh

```diff
@@ -0,0 +1,8 @@
+remove_temp_files() {
```

Comment:
`temp` is kind of generic, consider being more specific... e.g. `cleanFoundationBuildTempFiles`

Reply:
Good point